File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinTecsUserBackoffice SDK exists test.

using Xunit;

using BluefinTecsUserBackofficeSdk;

namespace BluefinTecsUserBackofficeSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinTecsUserBackofficeSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: swift/Tests/BluefinTecsUserBackofficeSdkTests/ExistsTest.swift
// BluefinTecsUserBackoffice SDK exists test.

import XCTest

@testable import BluefinTecsUserBackofficeSdk

final class ExistsTest: XCTestCase {
  func testMode() {
    let testsdk = BluefinTecsUserBackofficeSDK.testSDK(nil, nil)
    XCTAssertEqual(testsdk.mode, "test")
  }
}
